python/mlx/nn/losses.py

```diff
@@ -24,7 +35,42 @@ def cross_entropy(
     return _reduce(loss, reduction)
 
 
-def l1_loss(predictions: mx.array, targets: mx.array, reduction: str = "none") -> mx.array:
+def binary_cross_entropy(
+    inputs: mx.array, targets: mx.array, reduction: str = "none"
+) -> mx.array:
+    """
+    Computes the binary cross entropy loss between inputs and targets.
+
+    Args:
+        inputs (mx.array): The predicted inputs.
```

Comment:
Would be good to clarify this loss expects the post sigmoid inputs / inputs should be probabilities.